packages/app/components/common/ErrorBoundary.tsx

```diff
@@ -0,0 +1,43 @@
+import { apm } from "@elastic/apm-rum";
+import { Result } from "antd";
+import React, { ReactNode } from "react";
+
+interface EBState {
+  hasError: boolean;
+}
+// eslint-disable-next-line @typescript-eslint/ban-types
+export class ErrorBoundary extends React.Component<{}, EBState> {
+  constructor(props) {
+    super(props);
+    this.state = { hasError: false };
+  }
+
+  static getDerivedStateFromError(error): EBState {
+    // Update state so the next render will show the fallback UI.
+    return { hasError: true };
+  }
+
+  componentDidCatch(error, errorInfo): void {
+    // You can also log the error to an error reporting service
+    // logErrorToMyService(error, errorInfo);
+    console.log(error, apm)
+    if (window) {
+      apm.captureError(error);
```

Comment:
Error boundary will catch all misc errors and send them up to APM so we can actually figure out what the `unexpected errors` are